vulcanai2/models/basenetwork.py

```diff
@@ -188,99 +199,112 @@ def _init_trainer(self):
         self.criterion = self._get_criterion(self._criter_spec)
 
         self.valid_interv = 2*len(self.train_loader)
-        self.metrics = Metrics(self._num_classes)
         self.epoch = 0
 
 
-    def fit(self, train_loader, val_loader, epochs, retain_graph=False, valid_interv=None,
-                                                            print_iu=False):
+    def fit(self, train_loader, val_loader, epochs, retain_graph=False, valid_interv=None):
         
         self.train_loader = train_loader
         self.val_loader = val_loader
         self.epochs = epochs
         self.retain_graph = retain_graph
         if valid_interv:
             self.valid_interv = valid_interv
-        self.print_iu = print_iu
 
         self._init_trainer()
 
         for epoch in trange(self.epoch, epochs, desc='Epoch: ', ncols=80):
-            self.epoch = epoch
-            self.train_epoch()                    
+            train_loss, train_acc= self.train_epoch()
+            valid_loss, acc, avg_acc, iou, miou, conf_mat = self.validate()
+            tqdm.write("\n Epoch {}:\nTrain Loss: {:.6f} | Test Loss: {:.6f} | Train Acc: {:.2f}% | Test Acc: {:.2f}%".format(epoch, train_loss, valid_loss, train_acc*100, avg_acc*100))            
```

Comment:
no problem! may we could have some sort of option that can be added to the logger.

Reply:
well it's more of a consistency thing than anything else. It doesn't add more info to the user by converting it but all of our other metrics are in the range(0,1) so this would just make it a more cohesive set of metrics.

Reply:
I just realized we are sticking with a (0,1) range. I will change that.